examples/src/java/com/twitter/heron/examples/streamlet/FilesystemSinkTopology.java

```diff
@@ -115,7 +114,7 @@ public static void main(String[] args) throws Exception {
         .newSource(() -> {
           // This applies a "brake" that makes the processing graph write
           // to the temporary file at a reasonable, readable pace.
-          Utils.sleep(500);
```

Comment:
Not sure if this is necessary.

1. StreamletUtils is a class in examples directory. I feel it is reasonable for heron to provide a general function like sleep.
2. Utils looks like a low level object, correct? Why high level classes should avoid using it?

Reply:
I think one of attempts of these examples is to show that one can use just the high level api without relying on anything from the low level api.

Reply:
Make sense. Thanks.